rusoto/credential/tests/instance-profile-test.rs

```diff
@@ -0,0 +1,24 @@
+extern crate tokio_core;
+
+use rusoto_credential::{InstanceMetadataProvider, ProvideAwsCredentials};
+use std::time::Duration;
+use tokio_core::reactor::Core;
+
+#[test]
+#[ignore]
```

Comment:
This likely needs a comment as to why we're ignoring it: we only run this on a specific job since `cargo test --all` in the project root tries to run this integration test, which needs special setup.